modern/src/other/GeofencesPage.js

```diff
@@ -117,7 +118,9 @@ const GeofencesPage = () => {
             <label htmlFor="upload-gpx">
               <input accept=".gpx" id="upload-gpx" type="file" className={classes.fileInput} onChange={handleFile} />
               <IconButton edge="end" component="span" onClick={() => {}}>
-                <UploadFileIcon />
+                <Tooltip title={t('sharedUploadGeofence')} enterDelay={500} enterNextDelay={500}>
```

Comment:
Why do you have hardcoded delays everywhere?

Reply:
I can remove them.. the default delay is pretty short, and I figured that expert users might get sick of tooltips constantly popping up as they move their mouse around the screen.

Whereas beginners and people who don't know what a button does, are more likely to hover over a button for longer, when the tooltip will actually help them.

Reply:
If you think it's too long, we should change it globally, not per call.